core/src/display_object/movie_clip.rs

```diff
@@ -315,6 +321,75 @@ impl<'gc> MovieClip<'gc> {
         MovieClipWeak(GcCell::downgrade(self.0))
     }
 
+    pub fn new_import_assets(
+        activation: &mut Avm2Activation<'_, 'gc>,
+        movie: Arc<SwfMovie>,
+        parent: Arc<SwfMovie>,
+    ) -> Self {
+        let num_frames = movie.num_frames();
+
+        let loader_info = if movie.is_action_script_3() {
+            // The root movie doesn't have a `Loader`
+            // We will replace this with a `LoaderStream::Swf` later in this function
+            let loader_info =
+                LoaderInfoObject::not_yet_loaded(activation, movie.clone(), None, None, false)
+                    .expect("Failed to construct LoaderInfoObject");
+            let loader_info_obj = loader_info.as_loader_info_object().unwrap();
+            loader_info_obj.set_expose_content(activation.context.gc_context);
+            loader_info_obj.set_content_type(ContentType::Swf, activation.context.gc_context);
+            Some(loader_info)
+        } else {
+            None
+        };
+
+        let mc = MovieClip(GcCell::new(
+            activation.context.gc_context,
+            MovieClipData {
+                base: Default::default(),
+                static_data: Gc::new(
+                    activation.context.gc_context,
+                    MovieClipStatic::with_data(
+                        0,
+                        movie.clone().into(),
+                        num_frames,
+                        loader_info,
+                        activation.context.gc_context,
+                    ),
+                ),
+                tag_stream_pos: 0,
+                current_frame: 0,
+                audio_stream: None,
+                container: ChildContainer::new(movie.clone()),
+                object: None,
+                clip_event_handlers: Vec::new(),
+                clip_event_flags: ClipEventFlag::empty(),
+                frame_scripts: Vec::new(),
+                flags: MovieClipFlags::PLAYING,
+                drawing: Drawing::new(),
+                avm2_enabled: true,
+                avm2_use_hand_cursor: true,
+                button_mode: false,
+                last_queued_script_frame: None,
+                queued_script_frame: None,
+                queued_goto_frame: None,
+                drop_target: None,
+                hit_area: None,
+
+                #[cfg(feature = "timeline_debug")]
+                tag_frame_boundaries: Default::default(),
+                queued_tags: HashMap::new(),
+                attached_audio: None,
+                importer_movie: Some(parent.clone()),
+            },
+        ));
+
+        mc
+    }
+
+    pub fn get_importer_movie(&self) -> Option<Arc<SwfMovie>> {
```

Comment:
unused?
(if used, let's drop the `get_` prefix)

Reply:
Removed